community/values/src/main/java/org/neo4j/values/storable/TemporalValue.java

```diff
@@ -613,6 +617,15 @@ void assign( Builder<?> builder, AnyValue value )
                 throw new UnsupportedTemporalUnitException( "Not supported: " + name() );
             }
         },
+        offsetSeconds//<pre>
```

Comment:
What are these tags for?

Reply:
To be honest I just followed the pattern blindly. But @thobe should know maybe?

Reply:
The tags are for telling intellij to not autoformat that code.